docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,657 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "f486f849",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1. A Reuter consensus survey sees medical equipment group Radiometer reporting largely unchanged earnings when it publishes first half 19996/97 results next \u001b[31mWednesday\u001b[0m.\n",
+      "Given label: ORG, suggested label: O\n",
+      "\n",
+      "2. \u001b[31mLet\u001b[0m's march together,\" Scalfaro, a northerner himself, said.\n",
+      "Given label: LOC, suggested label: O\n",
+      "\n",
+      "3. Scottish \u001b[31mpremier\u001b[0m division after Saturday's matches:\n",
+      "Given label: MISC, suggested label: O\n",
+      "\n",
+      "4. \u001b[31mBut\u001b[0m 2 27/11/96 5,000 Burma\n",
+      "Given label: MISC, suggested label: O\n",
+      "\n",
+      "5. \u001b[31m1.\u001b[0m Fc Cologne 16 8 2 6 31 27 26\n",
+      "Given label: ORG, suggested label: O\n",
+      "\n",
+      "6. \u001b[31mNetwork\u001b[0m operators said the draft laws would hold them responsible for copyright infringements in the system and expose them to multi-billion-dollar liabilities.\n",
+      "Given label: MISC, suggested label: O\n",
+      "\n",
+      "7. -- \u001b[31mBangkok\u001b[0m newsroom (662) 652-0642\n",
+      "Given label: MISC, suggested label: LOC\n",
+      "\n",
+      "8. The lanky former Leeds United defender did not make his England debut until the age of 30 but eventually won 35 caps and was a key member of the \u001b[31m1966\u001b[0m World Cup winning team with his younger brother, Bobby.\n",
+      "Given label: MISC, suggested label: O\n",
+      "\n",
+      "9. The basket comprises Algeria's Saharan Blend, Indonesia's Minas, Nigeria's Bonny Light, Saudi Arabia's Arabian Light, \u001b[31mDubai\u001b[0m of the Uae, Venezuela's Tia Juana and Mexico's Isthmus.\n",
+      "Given label: MISC, suggested label: LOC\n",
+      "\n",
+      "10. Two ships loaded on the East \u001b[31mCoast\u001b[0m, three waited to load, six were due.\n",
+      "Given label: O, suggested label: LOC\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "entities = ['O', 'MISC', 'PER', 'ORG', 'LOC'] \n",
+    "\n",
+    "def color_sentence(sentence, word): \n",
+    "    start_idx = sentence.index(word) \n",
+    "    before, after = sentence[:start_idx], sentence[start_idx + len(word):]\n",
+    "    return '%s%s%s' % (before, colored(word, 'red'), after) \n",
+    "\n",
+    "def print_issue(issue, show_labels=True): \n",
+    "    i, j = index_to_tuple[issue] \n",
+    "    issue_label = entities[labels[issue]] \n",
+    "    predicted_label = entities[np.argmax(pred_probs[issue])] \n",
+    "    issue_word = given_words[i][j] \n",
+    "    print('%s' % color_sentence(sentences[i], issue_word)) \n",
+    "    if show_labels: \n",
+    "        print('Given label: %s, suggested label: %s\\n' % (issue_label, predicted_label)) \n",
```

Comment:
should we make this function part of the package?  
I.e. what if it lives in `cleanlab/token_classification/` as one of the user-facing functions?